### PR TITLE
Fixed #40

### LINKED ISSUE #40: croniter("30 22 * * sun-thu") fails ValueError
Hi,
the following test fails with __ValueError: [30 22 * * sun-thu] is not acceptable__:
Cron accepts this format.
```python
import unittest

from croniter import croniter
from datetime import datetime


class CroniterTest(unittest.TestCase):
    def sunday_to_thursday(self):
        base = datetime.today()
        exp = "30 22 * * sun-thu"
        my_iter = croniter(exp, base)

        next_execution = my_iter.get_next(datetime)
        print(next_execution.time())


if __name__ == '__main__':
    unittest.main()
```

### PR DESCRIPTION
**Pull request by steffenschroeder**
Issue: converting sun in range sun-thu transforms to int 0 which is recognized as empty string
Solution converting sun to string "0"

### CODE REVIEW

File: src/croniter/croniter.py [hunk 1]
```diff
@@ -81,10 +81,10 @@ def __init__(self, expr_format, start_time=time()):
                     (low, high, step) = m.group(1), m.group(2), m.group(4) or 1
 
                     if not any_int_re.search(low):
-                        low = self.ALPHACONV[i][low.lower()]
+                        low = str(self.ALPHACONV[i][low.lower()])
```
Comment (kiorky) [inline src/croniter/croniter.py]: please do not use str but format.
```
low = "{0}".format(self.ALPHACONV[i][low.lower()])
```

**Comment by kiorky** [comment]
when the little format thing will be done, i ll merge it.

ping kiorky on irc.freenode.net for the merge to be quickier ;)

**Comment by kiorky** [comment]
(the pr also need to be rebased)

**Comment by steffenschroeder** [comment]
Changed `str()` to `"{0}".format` and rebased the branch.
Let me know if I can do anything else

**Comment by kiorky** [comment]
it is good :)

**Comment by kiorky** [comment]
thx

**Comment by kiorky** [comment]
@steffenschroeder (the fix is released on pypi)

**Comment by steffenschroeder** [comment]
Thanks. I'll make use of it 😊

Mathieu Le Marec - Pasquet <notifications@github.com> schrieb am Mo., 1.
Juni 2015 12:40:

**Pull request merged**